web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -65,10 +66,19 @@ export const SidebarCardsList = (props: SidebarCardsListProps): ReactElement =>
           <hr className="margin-top-3 bg-cool-lighter" aria-hidden={true} />
           <div className="desktop:margin-bottom-1">
             <Accordion
+              ref={accordionRef}
               expanded={hiddenAccordionIsOpen}
               onChange={(): void => {
                 if (!hiddenAccordionIsOpen) {
                   analytics.event.for_you_card_unhide_button.click.unhide_cards();
+                  const element = accordionRef.current as unknown as HTMLElement;
+                  if (element) {
+                    scrollToTopOfElement(
+                      element.parentElement as HTMLDivElement,
+                      { isDesktop: isDesktopAndUp },
+                      200
+                    );
```

Comment:
[pebble] I liked what you had before, where this was contained in it's own function. I think there was a comment about keeping the logic in this file and not in a helper file, but that doesn't mean we have to just get rid of the function.

The other thing I really liked before was explicitly [giving a name to the value 200](https://github.com/newjersey/navigator.business.nj.gov/pull/6481#discussion_r1370315240). This gives additional context and purpose to the value.